### PR TITLE
dev/core##4146 Run civix upgrade on afform_html

### PR DESCRIPTION
**Pull request by eileenmcnaughton**
Overview
----------------------------------------
Should address https://github.com/civicrm/civicrm-core/pull/27565#issuecomment-1732034403

I have a feeling @totten has kinda abandoned this extension?

### CODE REVIEW

File: templates/CRM/Event/Form/Registration/ThankYou.tpl [hunk 1]
```diff
@@ -61,7 +61,7 @@
             <p>{ts}Your registration has been processed successfully.{/ts}</p>
             {if $is_email_confirm}
                 <p>{ts 1=$email}A registration confirmation email has also been sent to %1{/ts}</p>
-            {/if}
+            {/if}$email$email
```
Comment (larssandergreen) [inline templates/CRM/Event/Form/Registration/ThankYou.tpl:64]: Don't need this!

Comment (eileenmcnaughton) [reply templates/CRM/Event/Form/Registration/ThankYou.tpl:64]: ouch -

**Comment by totten** [comment]
Included as part of larger #27641.

**Comment by eileenmcnaughton** [comment]
yep - done!

**Comment by colemanw** [comment]
Actually it was done in https://github.com/civicrm/civicrm-core/pull/27617

**Pull request closed without merging**